vector/src/main/java/im/vector/app/features/call/webrtc/WebRtcCall.kt

```diff
@@ -280,13 +285,15 @@ class WebRtcCall(
         val peerConnectionFactory = peerConnectionFactoryProvider.get() ?: return
         val iceServers = mutableListOf<PeerConnection.IceServer>().apply {
             turnServerResponse?.let { server ->
-                server.uris?.forEach { uri ->
+                val useFallback = server.uris.isNullOrEmpty() && vectorPreferences.useFallbackTurnServer()
+                val serverList = if (useFallback) stringArrayProvider.getStringArray(R.array.fallback_ice_servers).toList() else server.uris
+                serverList?.forEach { uri ->
```

Comment:
Maybe a stupid question, but do we want to use the fallback if `turnServerResponse` is null? Here this will not be the case.

Reply:
Yes we should only use the fallback if the home server does not define any ice servers (consitent with web).

Reply:
Ok, so the code `turnServerResponse?.let { server ->` has to be updated IIUC.

Reply:
there, I've handled that now